spec/requests/cookies_spec.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "Cookies API", type: :request do
+  describe "PUT /cookies" do
+    it "handles analytics cookie acceptance" do
+      headers = { "ACCEPT" => "application/json", "CONTENT_TYPE" => "application/json" }
+      put "/cookies", params: '{"cookies_form":{"analytics_consent":"on"}}', headers: headers
```

Comment:
You don't need to put this in quotes, a hash will do just fine. You'll need to add spaces in the hash after the colon too.